pkg/api/types.go

```diff
@@ -2049,6 +2063,23 @@ type ServiceProxyOptions struct {
 	Path string
 }
 
+// OwnerReference contains enough information to let you identify an owning
+// object. Currently, an owning object must be in the same namespace, so there
+// is no namespace field.
+type OwnerReference struct {
+	// API version of the referent.
+	APIVersion string `json:"apiVersion,omitempty"`
```

Comment:
Do we really want group and version combined? Does that mean the generic garbage collector will have to watch every version of every object of every group?